Configuration/Settings.yaml

```diff
@@ -34,6 +34,15 @@ Neos:
           'Neos.Neos.UI:HostOnlyStyles':
             resource: '${"resource://" + Neos.Ui.StaticResources.compiledResourcePackage() + "/Public/Styles/HostOnlyStyles.css"}'
             position: 'start 900'
+          'WorkSans:200':
```

Comment:
Ehh you can't load the font like this, it ain't no webpack... ;)
You need to do smth like this CSS:
```
@font-face {
  font-family: "WorkSans";
  src: url("/...pathetofonts/work-sans-latin-200.woff") format("woff");
  font-weight: normal;
  font-style: normal;
}
```